reversion/templates/reversion/object_history.html

```diff
@@ -5,7 +5,7 @@
 {% block content %}
     <div id="content-main">
 
-        <p>{% blocktrans %}Choose a date from the list below to revert to a previous version of this object.{% endblocktrans %}</p>
+        <p>{% blocktrans %}Choose a date from the list below to view a previous version of this object and to revert to it.{% endblocktrans %}</p>
```

Comment:
I see what you're doing, but I'd say this description still implies that clicking the link will revert immediately!

How about "Choose a date from the list below to view a previous version of this object, and optionally revert to it."

Reply:
Of course, this will break all translations, so I'm not sure it's worth the clarification, since my translations are unlikely to get updated for a while.

Reply:
I updated the text. It's up to you to decide whether it's worth touching the translations.